File: 01/py/script1.py
fileName: str = "data.txt"

rawData: str = "";
with open(fileName) as file:
    for line in file.readlines():
        rawData += line
        pass
    pass

data: list[str] = rawData.split("\n")

maxCal: int = -1
currentCal: int = 0

for val in data:
    if val == "":
        if currentCal > maxCal:
            maxCal = currentCal
            pass
        currentCal = 0
        pass
    else:
        currentCal += int(val)
        pass
    pass

print(maxCal)

File: 2022/01/js/script1.ts
import { readFileSync } from "fs";

const data: string = readFileSync("data.txt").toString();
const cals: string[] = data.split("\n");

let maxCal: number = -Infinity;
let currentCals: number = 0;

cals.forEach((val: string) => {
  if (val === "") {
    if (currentCals > maxCal) {
      maxCal = currentCals;
    }

    currentCals = 0;
  } else {
    currentCals += parseInt(val);
  }
});
console.log(maxCal);
